### PR TITLE
Add ECS queue processing service example for TypeScript

### PR DESCRIPTION
**Pull request by iamhopaul123**
*Issue #, if available:*
None

*Description of changes:*
Add ECS queue processing service

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: typescript/ecs/ecs-service-with-queue-processing/index.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import ecs_patterns = require('@aws-cdk/aws-ecs-patterns');
+import cdk = require('@aws-cdk/core');
+
+class NihaoECS extends cdk.Stack {
```
Comment (rix0rrr) [inline typescript/ecs/ecs-service-with-queue-processing/index.ts]: Are you sure you want to publish `NihaoECS` as the stack name here?

File: typescript/ecs/ecs-service-with-queue-processing/index.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+     // Instantiate ECS Service with cluster and image
+    const ecsService = new ecs_patterns.QueueProcessingEc2Service(this, "Ec2Service", {
+      cluster,
+      memoryLimitMiB: 512,
+      image: ecs.ContainerImage.fromRegistry("amazon/amazon-ecs-sample"),
+    });
+
+    // Output the DNS where you can access your service
+    new cdk.CfnOutput(this, 'sqsQueueDNS', { value: ecsService.sqsQueue.queueName});
```
Comment (rix0rrr) [inline typescript/ecs/ecs-service-with-queue-processing/index.ts]: It's not a DNS though, it's a queue name.

File: typescript/ecs/ecs-service-with-queue-processing/index.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    const cluster = new ecs.Cluster(this, 'Ec2Cluster', { vpc });
+    cluster.addCapacity('DefaultAutoScalingGroup', {
+      instanceType: ec2.InstanceType.of(ec2.InstanceClass.T2, ec2.InstanceSize.MICRO)
+    });
+
+     // Instantiate ECS Service with cluster and image
+    const ecsService = new ecs_patterns.QueueProcessingEc2Service(this, "Ec2Service", {
+      cluster,
+      memoryLimitMiB: 512,
+      image: ecs.ContainerImage.fromRegistry("amazon/amazon-ecs-sample"),
```
Comment (rix0rrr) [inline typescript/ecs/ecs-service-with-queue-processing/index.ts]: This is not an image that does anything with the queue contents, right? 

What is the value of this example exactly?

File: typescript/ecs/ecs-service-with-queue-processing/index.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecs = require('@aws-cdk/aws-ecs');
+import ecs_patterns = require('@aws-cdk/aws-ecs-patterns');
+import cdk = require('@aws-cdk/core');
+
+class NihaoECS extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // For better iteration speed, it might make sense to put this VPC into
```
Comment (rix0rrr) [inline typescript/ecs/ecs-service-with-queue-processing/index.ts]: Doesn't seem like this comment needs to be in an example.

File: typescript/ecs/ecs-service-with-queue-processing/index.ts (hunk 3 above)
Comment (iamhopaul123) [reply typescript/ecs/ecs-service-with-queue-processing/index.ts]: Thanks for the review. It provides an example of how to create a queue processing service using the high level construct defined in the ecs pattern module. If that doesn't seem of any value to you, then I can close the pull request.

**Comment by SoManyHs** [comment]
Since this example is pretty old and doesn't give much extra information about this pattern outside of the regular API, I'm going to go ahead and close this.

**Pull request closed without merging**